tests/end2end/features/tabs.feature

```diff
@@ -255,6 +255,18 @@ Feature: Tab management
             - data/numbers/2.txt (active)
             - data/numbers/3.txt
 
+    Scenario: :tab-focus with 0
+        When I open data/numbers/1.txt
+        And I open data/numbers/2.txt in a new tab
+        And I open data/numbers/3.txt in a new tab
+        And I run :tab-focus 1
+        And I run :tab-focus 3
+        And I run :tab-focus 0
+        Then the following tabs should be open:
+            - data/numbers/1.txt (active)
```

Comment:
I find this surprising - wouldn't we expect `:tab-focus` with count (or index) 0 to focus the *last* tab now?